libs/common/package.json

```diff
@@ -55,6 +55,7 @@
     "express": "^4.21.2",
     "html-entities": "^2.5.2",
     "lodash.difference": "^4.5.0",
+    "lodash.orderby": "^4.6.0",
```

Comment:
can this not be extracted/used from the lodash package rather than importing this specific method?